tests/test_tensorrt_preprocess.py

```diff
@@ -0,0 +1,74 @@
+import os
+from functools import wraps
+
+import onnx
+import torch
+
+from mmcv.ops import nms
+from mmcv.tensorrt.tensorrt_utils import preprocess_onnx
+
+
+def remove_tmp_file(func):
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        onnx_file = 'tmp.onnx'
+        kwargs['onnx_file'] = onnx_file
+        result = func(*args, **kwargs)
```

Comment:
If the running of `func` fails, then `tmp.onnx` may not be deleted successfully. Could wrap it `try, except, finally`.

Reply:
Done.